pkg/storage/internalstorage/features.go

```diff
@@ -21,5 +21,5 @@ func init() {
 // defaultInternalStorageFeatureGates consists of all known custom internalstorage feature keys.
 // To add a new feature, define a key for it above and add it here.
 var defaultInternalStorageFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
-	AllowRawSQLQuery: {Default: false, PreRelease: featuregate.Alpha},
+	AllowRawSQLQuery: {Default: true, PreRelease: featuregate.Alpha},
```

Comment:
In general, we couldn't just change the default value of the Alpha feature gate to `true`.
We need to accompany an Alpha upgrade to Beta and create a separate issue to follow the upgrade, but right now it looks like it can't be upgraded to Beta.